cppcache/src/ExecutionImpl.cpp

```diff
@@ -131,7 +131,8 @@ std::shared_ptr<ResultCollector> ExecutionImpl::execute(
   serverOptimizeForWrite = ((attr->at(2) == 1) ? true : false);
 
   LOGDEBUG(
-      "ExecutionImpl::execute got functionAttributes from server for function = "
+      "ExecutionImpl::execute got functionAttributes from server for function "
+      "= "
```

Comment:
Dude, bad format here. The lines below can fit on this line. Please cleanup.

Reply:
Fixed. Good catch.